src/Conversion/ONNXToTOSA/Math/Gemm.cpp

```diff
@@ -188,19 +201,29 @@ class ONNXGemmOpLoweringToTOSA : public OpConversionPattern<ONNXGemmOp> {
     // If all check passed, we replace the GEMM by a FC operator
     Type resultType = getTypeConverter()->convertType(op.getResult().getType());
 
-    // If no bias is given to the GEMM operator, we create a 1D bias with all
-    // zeroes
-    if (C.getType().isa<mlir::NoneType>()) {
-      // Base C format on B[0] format
+    // Because the bias is not broadcastable for TOSA while it is for Onnx,
+    // we create an empty bias and use an add (broadcastable for tosa)
+    // afterwards
```

Comment:
```suggestion
    // afterwards.
```

nit: LLVM/MLIR comment standard suggests a full stop at the end of comments. Not sure if relevant, but we should probably capitalize `ONNX` and `TOSA` consistently.